charts/gitops-server/values.yaml

```diff
@@ -79,8 +79,8 @@ adminUser:
   # k8s. Requires `adminUser.create` and `adminUser.createSecret`.
   username: gitops-test-user
   # -- (string) Set the password for local admin user. Requires `adminUser.create` and `adminUser.createSecret`
-  # This needs to have been hashed using the bcrypt
-  # algorithm. E.g. `go install github.com/bitnami/bcrypt-cli@v1.0.2 && bcrypt-cli <<< $PASSWORD`
+  # This needs to have been hashed using a bcrypt
+  # algorithm. You can access one via our CLI with `gitops get bcrypt-hash`.
```

Comment:
Hm, "a bcrypt algorithm" sounds odd - it's one specific algorithm. Maybe "This needs to have been hashed using bcrypt"?